.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+env:
+  app: sidekiq_publisher
+
+jobs:
+  lint:
+    name: Lint (Ruby ${{ matrix.ruby }})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [ '2.6', '2.7', '3.0', '3.1', '3.2' ]
```

Comment:
We're still on Ruby 2.6 in this repo & presumably want to support through 3.2 -- though 3.1 and 3.2 would need alterations to pass